eemeter/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Version
-VERSION = (1, 2, 0)
+VERSION = (1, 2, 1)
```

Comment:
We'll bump the version number when we make a release, so it's best to revert this.